### DISCUSSION TITLE
Wiremod on Steam

### DISCUSSION THREAD

**Discussion by Noahfrechette** [wiremod/wire#2686] [General]
What Happened to WireMod on Steam?

**Comment by thegrb93** [comment]
It's hidden. Something to do with a eula I think? @Vurv78 @Anticept

**Comment by vurvdev** [comment]
See the discord #wire-announcements channel

**Comment by Noahfrechette** [comment]
They have a discord? can you link me to it?

**Reply by thegrb93** [reply]
There's a button/link if you scroll down on the main page https://github.com/wiremod/wire

**Comment by Anticept** [comment]
If it's hidden again let me know. I have to go to the account, hidden workshop item, and accept the eula.
